detection/notebooks/11_exploring_hyperparameters_on_azureml.ipynb

```diff
@@ -0,0 +1,618 @@
+{
```

Comment:
<p>I don't think we need to assign copy_tree to an empty var, so we can delete: "_ ="</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/ComputerVision/pull/303/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
I had to add the _ to suppress a lot of print-outs in the notebook